samples/monsterdata.json

```diff
@@ -5,5 +5,24 @@
     z: 3
   },
   hp: 300,
-  name: "Orc"
+  name: "Orc",
+  weapons:[
+    {
+      name: "axe",
+      damage:100
+    },
+    {
+      name: "bow",
+      damage:90
+    }
+  ],
+  // To ensure that the flatcc JSON parser in C can parse unions at maximum speed, 
```

Comment:
I don't think we want a comment like this in a sample, this is fairly implementation dependent.